docs/guides/akamai/get-started/parent-child-accounts/index.md

```diff
@@ -0,0 +1,154 @@
+
+The parent/child relationship doesn’t restrict the child account and its users from creating support tickets or signing up for managed services for their accounts.
+
+### Terminology
+
+#### Parent account
+In the context of your account type, you need to be familiar with the following terms.
+![Parent account terminology](parent_term.png)
+- *Parent account* - your, Akamai partner, account.
+- *Parent account user with full access* (parent account admin) - an Akamai partner account owner who has access to child accounts by default. They manage permissions for other admins and child account users with limited access. They may provide full access to a parent account user with limited access who then also becomes a parent account admin.
+- *Parent account users with limited access* - users of a parent account that don’t have access to child accounts by default. Only the parent account admin may grant them access to all child accounts the parent account has a contractual parent/child relationship with. To learn more, see [Enable access to child accounts for parent account users with limited access](#enable-access-to-child-accounts-for-parent-account-users-with-limited-access).
+
+#### Parent/child relationship
+In the context of the parent/child relationship, you need to be familiar with the following terms.
+![Parent/child relationship terminolog](parent_child_term.png)
+- *Parent account* - your, Akamai partner, account.
+- *Child account parent user* - a single user on a given child account that represents all parent account users with access to child accounts. It’s used to manage the child account. Depending on the access level and permissions provided by the child account admin, it may be a child account parent user with either full or limited access. To learn more, see [Switch accounts](#switch-accounts).
+- *Child account* - an account of your end customer.
+- *Child account users with full access* (child account admins) - users of the child account with full access to it. They manage permissions for all users on the account; other admins, child account users with limited access, and the child account parent user.  The only exception is billing, child account admins have read-only access to it and they can’t modify the child account parent user’s read/write permission to it.
+- *Child account users with limited access* - non-admin users of the child account whose permissions are managed by the child account admin.
+
+One parent account may manage many child accounts.
+
+### Enable access to child accounts for parent account users with limited access
+
+As a parent account admin, you can grant access to the child account for parent account users with limited access.
+
+On the User Permissions page, you have the additional general permission - **Enable child account access**.
```

Comment:
I know this is petty, but the hyphen here should be replaced with a colon, or you could rephrase this to:

"On the User Permissions page, you have access to **Enable child account access**."